docs/iot/guides/update-control-api.md

```diff
@@ -0,0 +1,134 @@
+# Tizen Update Control Native API
```

Comment:
@sabeena21 When a new page is added,
PLEASE don't forget to guide to edit the TOC.md file, too.

@SunmLee Please edit docs/iot/TOC.md.
This file is to list up the menu tree, the location, at where you are going to place this page.

Reply:
@mijin85cho, okay sure